pkg/lint/policy_builtins.go

```diff
@@ -422,4 +422,17 @@ var builtinRules = map[string]validators{
 			}, nil
 		},
 	},
+	"pipeline-has-start-date": {
+		Pipeline: func(pipeline *pipeline.Pipeline) ([]*Issue, error) {
+			if strings.TrimSpace(pipeline.StartDate) != "" {
```

Comment:
are you sure we dont have a default start date here ?

Reply:
Yeah. It can be left empty.